src/rviz/message_filter_display.h

```diff
@@ -62,14 +63,19 @@ Q_OBJECT
                                                "Prefer UDP topic transport",
                                                this,
                                                SLOT( updateTopic() ));
+      queue_size_property_ = new IntProperty( "Queue Size", 10,
+                                          "Size of the tf message filter queue. It usually needs to be set at least as high as the number of sonar frames.",
```

Comment:
There's a copy-paste reference here. Please update the comment to something [like this](https://github.com/ros-visualization/rviz/blob/7970ba08cee3810cfa1609c3b0f5136970eb2f7c/src/rviz/default_plugin/marker_display.cpp#L70..L73) (but without Marker references)